src/main/webapp/components/Navigation.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+/**
+ * Main List Element to be displayed on the Navigation.
+ *
+ * @param {href: a URL String to a resource, innerText: The display text}
```

Comment:
In jsdoc `{}` after `@param` is interpreted not as free text, but as a type tag with a specific syntax:
https://jsdoc.app/tags-param.html#names-types-and-descriptions
https://jsdoc.app/tags-type.html
```suggestion
 * @param {href: a URL String to a resource, innerText: The display text}
```
```suggestion
 * @param {href: string, innerText: string} props (documentation)
```

Since this "Option type" described here is referenced below, you could use a `@typedef` to define its properties and refer to the name of that type definition here and below. You can see some `@typedef` examples here: https://devhints.io/jsdoc

Reply:
Updated in relation to [Documenting properties of values in an array](https://jsdoc.app/tags-param.html#parameters-with-properties)